src/models/dashboard.js

```diff
@@ -35,29 +36,51 @@ export default {
     },
     *fetchControllers({ payload }, { call, put }) {
       const controllers = yield call(queryControllers, payload);
-
+      console.log(controllers);
       yield put({
         type: 'getControllers',
         payload: controllers,
       });
     },
+    *fetchPrivateResults({ payload }, { put }) {
```

Comment:
What are your thoughts on removing the context of authentication and instead including ownership to naming of actions such as `fetchUserResults` in this case?

Reply:
The action was redundant so removed it. Thank you for noticing